packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -685,15 +691,15 @@ export const activate = async (extensionContext: vscode.ExtensionContext) => {
       .getConfiguration(SFDX_CORE_CONFIGURATION_NAME)
       .get<boolean>(ENABLE_SOBJECT_REFRESH_ON_STARTUP, false);
 
-    if (sobjectRefreshStartup) {
-      initSObjectDefinitions(
-        vscode.workspace.workspaceFolders[0].uri.fsPath
-      ).catch(e => telemetryService.sendException(e.name, e.message));
-    } else {
-      checkSObjectsAndRefresh(
-        vscode.workspace.workspaceFolders[0].uri.fsPath
-      ).catch(e => telemetryService.sendException(e.name, e.message));
-    }
+    initSObjectDefinitions(
+      vscode.workspace.workspaceFolders[0].uri.fsPath,
```

Comment:
Can workspaceFolders be empty here?  Nothing sadder then the ol `unable to read uri of undefined` or whatever

Reply:
haha no, it can't. We do the validation a couple lines before